auth/SBASKey.m

```diff
@@ -36,19 +36,19 @@
                     obj.key = zeros(16, 1, 'uint8');
                     obj.signature = uint8(randi([0, 255], 64, 1));
                 otherwise
-                    error('Key:InvalidKeyLevel', 'Received key level %i, which is not 1 or 2.', level);
+                    error('SBASKey:InvalidKeyLevel', 'Received key level %i, which is not 1 or 2.', level);
             end
 
             if nargin > 4
```

Comment:
This is a protection against expansion...as I add features...this will remind the engineer to come back to this line and update this switch logic....

this is why every if/switch statement should have full coverage...even if some of the coverage results in an error...otherwise the engineer will forget about this logic when adding features.